changelog/1531.trivial.rst

```diff
@@ -1,2 +1,3 @@
-Reduced execution time by avoiding calculations with units while keeping unit validation.
-Added `plasmapy/utils/units_definitions.py` to precompute units.
+Reduced execution time by avoiding calculations with units while keeping
+unit validation. Added :file:`plasmapy/utils/units_definitions.py` to
+precompute units.
```

Comment:
This entry does not make sense on it's own.  There is no context.  Reduced execution timing of what?  What is `units_definitions.py`?  Also, this seems like two separate changelog entries.